MyCryptoHQ/MyCrypto
# Grey out disabled swap options

Closes #2025

### Description

Added a method to the ShapeShift service to also send unavailable coins and tokens.

### Changes

* The SwapDropdown now shows unavailable coins and tokens at the end of the list, and they are disabled.

### Steps to Test

1. Open swap.
2. Click the dropdown and see greyed out, disabled coins or tokens at the end of the list.

### Screenshots

<img width="604" alt="screen shot 2018-07-06 at 1 22 10 pm" src="https://user-images.githubusercontent.com/10479826/42394412-a7fd4cf6-811f-11e8-84c1-b091a678eb5d.png">
